app/assets/javascripts/components/projects/ProjectBoard.js

```diff
@@ -64,6 +65,12 @@ export const ProjectBoard = ({
     fetchProjectBoard(projectId);
   }, [fetchProjectBoard, projectId]);
 
+  useEffect(() => {
+    subscribeToProjectChanges({id: projectId}, () => {
```

Comment:
```ts
  subscribeToProjectChanges({ id }, () => {
    fetchProjectBoard(projectId);
```
:nail_care:

Reply:
I added the spaces. However, I think that ```id:projectId``` fits better for this specific line - ```projectId``` is a prop.